SCAutolib/cli_commands.py

```diff
@@ -165,7 +169,47 @@ def cleanup(ctx):
     exit(ReturnCode.SUCCESS.value)
 
 
+@click.command()
+@click.option('--init',
+              required=False,
+              default=False,
+              is_flag=True,
+              help="Initialize GUI for manual testing")
+@click.option("--assert-text",
+              required=False,
+              default=None,
+              show_default=True,
+              help="Check if a word is found on the screen ")
+@click.option("--click-on",
+              required=False,
+              default=None,
+              show_default=True,
+              help="Click on object containing word")
+@click.option("--check-home-screen",
+              required=False,
+              is_flag=True,
+              default=False,
+              show_default=True,
+              help="Check if screen appears to be the home screen")
+@click.pass_context
+def gui(ctx, init, assert_text, click_on, check_home_screen):
```

Comment:
In general, I am wondering what is the best approach. Maybe running one thing at a time is not that optimal and it needs a lot of refactoring. Maybe a better approach will be to "construct" a dynamic test through arguments like the tests in https://github.com/redhat-qe-security/SC-tests/blob/main/Graphical/.  and then follow somehow the arguments the way they have given. Finally, we can keep the state somehow so as to run multiple commands and initialize it with --init and exit with --done or something. What is your opinion on that?

Reply:
That is why I thought to have some init and finish flags and when the init one was specified to not run __exit__ or something like that.